tests/features/content_types/content_type_administration.feature

```diff
@@ -0,0 +1,48 @@
+@api
+Feature: Content type administration features
+  In order to easily structure the content of the European Commission
+  As an administrator
+  I want to be able to create, customize and delete content types
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+
+  Scenario: As administrator, I create a custom content type translated through
+    entity translation, using workbench moderation and having 2 fields "Body" and
+    "Select option". I should be able to create a content with it
+    When I go to "admin/structure/types/add"
+    And I fill in "Name" with "Groovy type"
+    And I fill in "Machine-readable name" with "groovy_type"
+    And I fill in "Description" with "A Groovy type for a groovy test"
+    And I uncheck the box "Published"
+    And I uncheck the box "Promoted to front page"
+    But I check the box "Create new revision"
+    And I check the box "Enable moderation of revisions"
+    And I select the radio button "Enabled, with field translation" with the id "edit-language-content-type-4"
+    And I press the "Save and add fields" button for saving the "groovy_type" content type
+    Then I should be on "admin/structure/types/manage/groovy-type/fields_en"
+    And I should see the success message "The content type Groovy type has been added."
+    And I should see "Body"
+    When I fill in "edit-fields-add-new-field-label" with "Select an option"
+    And I fill in "edit-fields-add-new-field-field-name" with "select_an_option"
```

Comment:
I tried several ways to do it with @javascript and that failed all the time. Taking into account the time on it, I have chosen this way on purpose.